helm_deploy/create-and-vary-a-licence-api/templates/cronjob-data-extractor-analytics.yaml

```diff
@@ -55,4 +55,10 @@ spec:
                   value: eu-west-2
                 - name: SAVE_EVENTS_LOG
                   value: "true"
+              resources:
+                requests:
+                  memory: 2G
+                  cpu: "1"
+                limits:
+                  memory: 4G
```

Comment:
copied from one of CRDS projects - we can reduce this if appropriate, just have to see how it performs in prod